Updating the 'Job Hunting' section and fixing a few spelling errors

scouttyg opened the pull request.

scouttyg committed:
Added in TrueJob link to job hunting resources

scouttyg committed:
Fixed a small spelling error on the page

scouttyg commented on index.html:
```diff
@@ -372,7 +372,7 @@
                 </li>
                 <li>
                     <a href="http://fordschool.umich.edu/events/upcoming">Gerald R. Ford School of Public Policy Science, Technology, and Public Policy Program Events</a>
-                    <p>The University of Michigan's Gerald R. Ford School of Public Policy's science and technology events, includes public events. (You'll need to filter their calendar for the approrpiate topics and public access.)</p>
```
Just a small spelling error here -- approrpiate -> appropriate

scouttyg commented on index.html:
```diff
@@ -528,8 +528,12 @@
                 <p>Talent consultants and recruiters/matchmakers focused on supporting Michigan technology companies with their talent needs.</p>
               </li>
               <li>
-                <a href="https://angel.co/michigan">AngleList - Ann Arbor Startups</a>
```
Another small spelling correction - AngleList -> AngelList

scouttyg commented on index.html:
```diff
@@ -528,8 +528,12 @@
               <li>
-                <a href="https://angel.co/michigan">AngleList - Ann Arbor Startups</a>
-                <p>A listing of Ann Arbor startups, including job openings.</p>
+                <a href="https://angel.co/michigan">AngelList - Michigan Startups</a>
```
The link for AngelList had the text "Ann Arbor Startups", but linked to https://angel.co/michigan, which is all Michigan startups. I updated the text here to instead say 'Michigan Startups', but we could also link to https://angel.co/ann-arbor instead and keep the old text.

jshwlkr replied:
Yeah, could you just switch the link and resubmit the merge request, please?

scouttyg commented on index.html:
```diff
@@ -528,8 +528,12 @@
+                <p>A listing of Michigan startups, including job openings.</p>
+              </li>
+              <li>
+                <a href="https://www.truejob.com/locations/ann-arbor/jobs">TrueJob - Ann Arbor Startups</a>
```
@mwkling and I created [TrueJob](http://www.truejob.com/) here in Ann Arbor, and we have the biggest list of Ann Arbor startup jobs of any other site.

scouttyg committed:
Fixed AngelList link - AngelList was spelled AngleList and was linked to just Michigan startups, not Ann Arbor startups

scouttyg commented on index.html:
```diff
@@ -528,9 +528,13 @@
               </li>
               <li>
-                <a href="https://angel.co/michigan">AngleList - Ann Arbor Startups</a>
+                <a href="https://angel.co/ann-arbor">AngleList - Ann Arbor Startups</a>
```
Fixed this commit so instead we don't change the text, we instead change the link!

scouttyg commented on index.html:
```diff
@@ -528,9 +528,13 @@
                 <p>A listing of Ann Arbor startups, including job openings.</p>
               </li>
+              <li>
+                <a href="https://www.truejob.com/locations/ann-arbor/jobs">TrueJob - Ann Arbor Startups</a>
```
@mwkling and I created TrueJob here in Ann Arbor, and we have the biggest list of Ann Arbor startup jobs of any other site.

scouttyg commented:
Updated the pull request! Our old comments were deleted/hidden so I added a few back, but I made it so instead of changing the text for AngelList to Michigan we instead link to the correct AngelList link for Ann Arbor
